### DISCUSSION TITLE
Please help me to restore my keyboard light

### DISCUSSION THREAD

**Discussion by Doublevland** [T-Troll/alienfx-tools#280] [Q&A]
Please help me understand if I can do something to make the lighting work again.
After an update to windows 11, my Awcc shows me the error: "no supported alienfx devices were detected". for 2 months I've been looking for different ways to somehow restore the keyboard lighting on my Dell G17 7700, but without any results, I've lost hope already.
I got here, and I was very happy that you created a possibility to replace awcc. I read the entire AlienFX-Tools wiki, but unfortunately after starting alienfx-gui, I don't see a tab with lights, there are only Profiles and Settings. I note that AWCC is not currently installed. Please help me very much.

**Comment by T-Troll** [comment]
Oh... This means no hardware light devices detected.
You can try to open CMD and run `alienfx-cli probe` (answer "N" for the question). It will reveal the list of all compatible devices found into the system.
For G-series, there can be some different configurations. Some of them doesn't have RGB.

UPD: According to spec [here](https://www.dell.com/ae/p/g-series-17-7700-laptop/pd), you should have both RGB keyboard and backlit, like 5500. In case it is not exposed into the system, first check device manager about broken devices (you need one with VID_187c&PID_0550). If it not found - you have a hardware issue. You can try to restart light system (read tools readme "issues" about how to do it), but i case this didn't help - contact Dell, i can't help with this.

**Reply by Doublevland** [reply]
Thanks for the reply. I used the keyboard light until I switched to windows 11 and updated the bios through Dell Update, after that, the light disappeared, and Awcc tells me that > "no supported alienfx devices were detected".
- I tried to open CMD and run alienfx-cli probe > response > Dell API not found, Low-level devices not found
Both low-level and high-level devices not found, exiting!

what about the Device manager, I know for sure that I had a problem with a HID Driver, I didn't know what to do with it and I installed Iobit Driver Booster, it automatically loaded and updated all the drivers.
Now I have a problem with a DellInstrumentation Device driver, which I also don't know what it is.

**Reply by Doublevland** [reply]
I found something similar in my Device Manager, maybe it's because of this?

HID keyboard Device > Currently, this hardware device is not connected to the computer. (Code 45) >  

Device settings for HID\VID_046D&PID_C534&MI_00\7&2807c94&0&0000 were migrated from previous OS installation.

Last Device Instance Id: HID\VID_046D&PID_C534&MI_00\7&2807c94&0&0000
Class Guid: {4d36e96b-e325-11ce-bfc1-08002be10318}
Location Path: 
Migration Rank: 0x0
Present: true


https://prnt.sc/wpeXF13dnwIZ

https://prnt.sc/V64_YEIACCZr

**Reply by T-Troll** [reply]
So yes, you lost USB light device completely.
No, this device is wrong, as i said, it should have VID_187C.
This can be a result of incorrect BIOS or AWCC update (i meet this before at old 17R4).

What to do.... First, try to reset light system (see in readme how to do it) - hand device not exposed into Windows as well.
But if it not restarts after it - chip EC broken, and i don't know how to reveal it.

**Reply by T-Troll** [reply]
PS: I see you have a lot of "vendor-defined device"s. Check them - maybe one of it is light device, but lost correct VID/PID - both my tools and AWCC will miss it in this case ^_^.
